cmd/geth/chaincmd.go

```diff
@@ -146,9 +149,19 @@ func importChain(ctx *cli.Context) error {
 	}()
 	// Import the chain
 	start := time.Now()
-	if err := utils.ImportChain(chain, ctx.Args().First()); err != nil {
-		utils.Fatalf("Import error: %v", err)
+
+	if len(ctx.Args()) == 1 {
+		if err := utils.ImportChain(chain, ctx.Args().First()); err != nil {
+			utils.Fatalf("Import error: %v", err)
+		}
+	} else {
+		for _, arg := range ctx.Args() {
+			if err := utils.ImportChain(chain, arg); err != nil {
+				log.Error("Import error", "err", err)
```

Comment:
I could, but the 'internal' messaging is already quite informative: 
``` 
INFO [03-08|09:04:47] Importing blockchain                     file=0001.rlp
ERROR[03-08|09:04:47] Import error                             err="open 0001.rlp: no such file or directory"
INFO [03-08|09:04:47] Importing blockchain                     file=0002.rlp
ERROR[03-08|09:04:47] Import error                             err="open 0002.rlp: no such file or directory"
```
Ping again if you want it even so